src/lib/transition_handler/catchup_scheduler.ml

```diff
@@ -48,7 +52,9 @@ module Make (Inputs : Inputs.S) = struct
               timeouts. *)
     ; parent_root_timeouts: unit Time.Timeout.t State_hash.Table.t
     ; breadcrumb_builder_supervisor:
-        ( (External_transition.Verified.t, State_hash.t) With_hash.t
+        ( ( External_transition.Verified.t Envelope.Incoming.t
```

Comment:
this type is too big and verbose. Can we have a type alias for it. Like `type enveloped_transition = ...`